python/lsst/afw/display/rgb.py

```diff
@@ -104,35 +238,116 @@ def __init__(self, min, range, Q):
         self._soften = Q/float(range);
```

Comment:
It's not part of your changes, but could you remove the "if False:" statement above?